### PR TITLE
Linter

### LINKED ISSUE #14: Add linter stage
ref: https://github.com/SSARCandy/Coherent-Line-Drawing/blob/master/linter.sh

### PR DESCRIPTION
**Pull request by SSARCandy**
close #14

### CODE REVIEW

File: linter.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+fi
+
+if command -v clang-format > /dev/null; then
+	CLANG_FORMATTER=$(command -v clang-format)
+else
+	echo -e "Missing clang-format, please install it with: \n    ${COLOR_BLUE}sudo apt-get install clang-format${COLOR_REST}"
+	exit 1
+fi
+
+file_list=$(find ini \( -name '*.h' -or -name '*.cpp' \))
```
Comment (hhschu) [inline linter.sh]: Do you not want to include the tests as well?

**Review by hhschu — COMMENTED** [review]
Does clang-format come with g++ or is it already installed in GitHub workflow's environment?

File: linter.sh (hunk 1 above)
Comment (SSARCandy) [reply linter.sh]: let's include them

**Comment by SSARCandy** [comment]
> Does clang-format come with g++ or is it already installed in GitHub workflow's environment?

It would be safer to install it, will fix it.

**Review by hhschu — APPROVED** [review]

**Pull request merged**